code/modules/tgs/core/datum.dm

```diff
@@ -11,6 +11,15 @@ TGS_DEFINE_AND_SET_GLOBAL(tgs, null)
 	src.event_handler = event_handler
```

Comment:
This file got renamed to avoid files with the same name in the codebase

Reply:
You better take care of it then.

Reply:
Reminder, this branch has maintainer edits enabled

Reply:
> Reminder, this branch has maintainer edits enabled

Oh I didnt see this and already made another PR